CONTRIBUTION_LADDER.md

```diff
@@ -0,0 +1,102 @@
+# Contribution Ladder
+
+---
+* [Contributor](#contributor)
+* [How to become a contributor](#how-to-become-a-contributor)
+* [Maintainer](#maintainer)
+* [How to become a maintainer](#how-to-become-a-maintainer)
+* [Admin](#admin)
+* [How to become an admin](#admin)
+---
+
+Our ladder defines the roles and responsibilities on this project and how to
+participate with the goal of moving from a user to a maintainer. You will need
+to gain people's trust, demonstrate your competence and understanding, and meet
+the requirements of the role.
+
+## Community Members
+
+Everyone is a community member! 😄 You've read this far so you are already ahead. 💯
+
+Here are some ideas for how you can be more involved and participate in the community:
+
+* Comment on an issue that you’re interested in.
+* Submit a pull request to fix an issue.
+* Report a bug.
+* Share a bundle that you made and how it went.
+* Come chat with us in [Slack][slack].
+
+They must follow our [Code of Conduct](CODE_OF_CONDUCT.md).
+
```

Comment:
```suggestion

[slack]: https://porter.sh/community#slack

```

reference to slack link

Reply:
Thanks! That link definition got lost. 👍